multiapps-controller-core/src/main/resources/org/cloudfoundry/multiapps/controller/core/db/changelog/db-changelog-1.110.0-core.xml

```diff
@@ -0,0 +1,265 @@
+        <createSequence sequenceName="configuration_subscription_sequence" startValue="1" incrementBy="1" />
+        <createTable tableName="configuration_subscription">
+            <column name="id" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="configuration_subscription_pk" />
+            </column>
+            <column name="mta_id" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="space_id" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="app_name" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="filter" type="CLOB">
+                <constraints nullable="false" />
+            </column>
+            <column name="module_id" type="VARCHAR(255)">
+                <constraints nullable="true" />
+            </column>
+            <column name="module" type="CLOB">
+                <constraints nullable="false" />
+            </column>
+            <column name="resource_id" type="VARCHAR(255)">
+                <constraints nullable="true" />
+            </column>
+            <column name="resource_name" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="resource_properties" type="CLOB">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+        <addUniqueConstraint tableName="configuration_subscription"
+            columnNames="mta_id, app_name, space_id, resource_name" constraintName="configuration_subscription_uq" />
+    </changeSet>
+
+    <changeSet author="sap.com" id="create_table_historic_operation_event">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <sequenceExists sequenceName="historic_operation_event_sequence" />
+            </not>
+            <not>
+                <tableExists tableName="historic_operation_event" />
+            </not>
+        </preConditions>
+        <createSequence sequenceName="historic_operation_event_sequence" startValue="1" incrementBy="1" />
+        <createTable tableName="historic_operation_event">
+            <column name="id" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="historic_operation_event_pk" nullable="false" />
+            </column>
+            <column name="process_id" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="event" type="VARCHAR(100)">
+                <constraints nullable="false" />
+            </column>
+            <column name="timestamp" type="TIMESTAMP">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+        <createIndex tableName="historic_operation_event" indexName="idx_historic_operation_event_process_id ">
```

Comment:
Previously we have added java classes which perform async index creation with `CONCURRENTLY` keyword which create index without locking the whole table. Does it mean now liquibase will use this feature of PostgreSQL to create indexes without locking whole tables?

Reply:
Nope. Indexes will now be created non-concurrently, but this is fine since these changesets will only be executed if the table/index/sequence doesn't already exist.